spec/features/admin_item_management_spec.rb

```diff
@@ -47,6 +47,21 @@
       expect(page).to have_content("Successfully Updated")
     end
 
+    it "cannot update an item to match an existing item's attributes" do
+      item_2 = create(:item, name: "Nacho Toast", description: "not your toast", price: 3.00)
```

Comment:
Useless assignment to variable - `item_2`. Did you mean `item`?<br>Line is too long. [93/80]